sentry-core/src/envelope.rs

```diff
@@ -0,0 +1,148 @@
+use std::io::Write;
+
+use crate::protocol::Event;
```

Comment:
Should envelope not also live there?

Reply:
you mean the `protocol` module? Not really sure, but we can move it there later on as well.

Reply:
yeah, meant the protocol module.  I guess need to decide on this before releasing it though